packages/theme_tailor_figma_loader/lib/src/utils/utils.dart

```diff
@@ -0,0 +1,51 @@
+import '../models/app_typography.dart';
+
+String toColor(String str) {
+  return 'Color(0xFF${str.substring(1, 7).toString().toUpperCase()})';
+}
+
+String typographyParser(AppTypography typography) {
+  final buffer = StringBuffer();
+
+  if (typography.fontFamily != 'none') {
+    buffer.write('\n    fontFamily: \'${typography.fontFamily}\',');
+  }
```

Comment:
Can remove whitespace formatting here and below (Will be autoformatted when file is generated)